src/system/PredefinedProcedure.h

```diff
@@ -12,6 +12,8 @@
 #include <vector>
 
 #include "data/ast/ProcedureNode.h"
+#include "data/ast/ASTContext.h"
+#include "data/ast/NodeVisitor.h"
```

Comment:
What are these includes used for?

Reply:
Removed. This was used in an early version.